server/controllers/stock/reports/stock/inline_movements_report.js

```diff
@@ -0,0 +1,57 @@
+const {
+  _, ReportManager, Stock, formatFilters, pdfOptions, STOCK_INLINE_MOVEMENTS_REPORT_TEMPLATE,
+} = require('../common');
+
+/**
+ * @method stockInlineMovementsReport
+ *
+ * @description
+ * This method builds the stock movements report as either a JSON, PDF, or HTML
+ * file to be sent to the client.
+ *
+ * GET /reports/stock/inline-movements
+ */
+async function stockInlineMovementsReport(req, res, next) {
+  let display = {};
+  const optionReport = _.extend(req.query, pdfOptions, {
+    filename : 'TREE.STOCK_INLINE_MOVEMENTS',
+    csvKey : 'rows',
+    renameKeys : false,
+  });
+
+  // set up the report with report manager
+  try {
+    if (req.query.displayNames) {
```

Comment:
Why remove the `try/catch` ? There is `await Stock.getMovements(null, req.query);` and `await report.render(data);` which may thrown errors

Reply:
Sorry, the `try`/`catch` around the `req.query.displayNames`.  Basically, remove the entire block, since it isn't functional and a future developer shouldn't think that code does something important if it doesn't do anything at all.